pymaid/upload.py

```diff
@@ -580,6 +590,20 @@ def upload_neuron(x, import_tags=False, import_annotations=False,
 
         resp['link_response'] = ln_resp
 
+        if import_tags and getattr(x, 'connector_tags', {}):
+            # Map old to new connectors
+            cn_tags = {t: [cn_map[n] for n in v] for t, v in x.connector_tags.items()}
+            # Invert connector tag dictionary: map connctor ID -> list of tags
+            ctags = {}
+            for t in cn_tags:
+                ctags.update({n: ctags.get(n, []) + [t] for n in cn_tags[t]})
+
+            resp['connector_tags'] = add_tags(list(ctags.keys()),
+                                              ctags,
+                                              'CONNECTOR',
+                                              override_existing=True,
```

Comment:
Do we want `override_existing=True` here? I.e. do we want to _add_ or _set_ the tags?

Reply:
Having `override_existing=True` here matches the behavior for treenode tags. And independent of the current behavior for tags, yea, I think we want to set them, not just add new tags. What do you think?

Reply:
I agree!